js/src/wrappers/ai-sdk-v1.ts

```diff
@@ -1,4 +1,11 @@
-import { ContentPart, Message, Tools } from "@braintrust/core/typespecs";
+import {
```

Comment:
this reminds me that (but lower prio.) the AI proxy also depends on the core/js:

https://github.com/braintrustdata/braintrust/blob/4fb59f67c1d4078ae2bd35e5e6a038832ca6c468/api-ts/src/proxy/proxy.ts#L29